onnx/test/reference_evaluator_test.py

```diff
@@ -24,8 +24,16 @@
 
 import numpy as np
 import parameterized
+from numpy import __version__ as npver
 from numpy.testing import assert_allclose
 
+try:
+    from packaging.version import parse as version
+except ImportError:
+    from distutils.version import (  # noqa: N813
+        StrictVersion as version,
+    )
```

Comment:
```suggestion
```

Reply:
We should not be using distutils any more because it is deprecated. packaging should be in our dev requirement

Reply:
I started with this but end up with current code.

Reply:
I see no difference

Reply:
restructured the code as suggested